### PR TITLE
Increment sequence number upon edition

### PR DESCRIPTION
**Pull request by Sakshisaraswat**
Increment sequence number upon edition.a variable y is taken and value of self.todo['SEQUENCE'] is taken in this variable . value of y is checked and if it is none then value of y is set to 0 and if not then value of sequence is incremented by one

### CODE REVIEW

**Comment by untitaker** [comment]
Looks good to me! Please add a testcase in `test_model.py`, a changelog entry and yourself to AUTHORS. Thanks!

File: todoman/model.py [hunk 1]
```diff
@@ -322,7 +322,10 @@ def save(self, list_=None):
         list_ = list_ or self.list
         path = os.path.join(list_.path, self.filename)
         assert path.startswith(list_.path)
-
+        y = self.todo.get('SEQUENCE')
```
Comment (hobarrera) [inline todoman/model.py]: I'd rather this variable be called `seq` or `sequence`, rather than `y` (eg: something more descriptive).

**Comment by Sakshisaraswat** [comment]
ok i will make changes

**Comment by Sakshisaraswat** [comment]
Please tell me how to write test cases?

File: todoman/model.py [hunk 2]
```diff
@@ -322,7 +322,10 @@ def save(self, list_=None):
         list_ = list_ or self.list
         path = os.path.join(list_.path, self.filename)
         assert path.startswith(list_.path)
-
+        sequence = self.todo.get('SEQUENCE')
+        if sequence is None:
+            sequence = 0
```
Comment (hobarrera) [inline todoman/model.py]: These three lines can actually all be simplified to `sequence = self.todo.get('SEQUENCE', 0)`. What it does is return the sequence, of if it's `None`, return the second value (which is `0`).

**Comment by hobarrera** [comment]
I took the liberty of fixing merge conflicts, but the prior review is still pending.

**Comment by hobarrera** [comment]
Thanks! :+1:

**Pull request merged**